src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
```

Comment:
The width of the product image should be set to 100% to make it responsive. Using fixed width can make it less flexible when the card component is used in different layouts. Consider using `width: 100%;` to ensure that the image scales with the card.